File: src/main/java/um/haberes/report/model/haberes/core/DesignacionTipoDto.java
package um.haberes.report.model.haberes.core;

import java.math.BigDecimal;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class DesignacionTipoDto {

    private int designacionTipoId = 0;

    private String nombre = "";

    private BigDecimal horasSemanales = BigDecimal.ZERO;

    private BigDecimal horasTotales = BigDecimal.ZERO;

    private int simples = 0;

}

File: src/main/java/um/haberes/report/kotlin/dto/DesignacionTipoDto.kt
package um.haberes.report.kotlin.dto

import java.math.BigDecimal

data class DesignacionTipoDto(

    var designacionTipoId: Int = 0,
    var nombre: String = "",
    var horasSemanales: BigDecimal = BigDecimal.ZERO,
    var horasTotales: BigDecimal = BigDecimal.ZERO,
    var simples: Int = 0

)
